jerry-debugger/jerry-client-ws.py

```diff
@@ -169,6 +172,13 @@ def insert_breakpoint(self, args):
         else:
             set_breakpoint(self.debugger, args)
 
+    def disable_args(self, args):
+        if args != "":
+            print("Error: No argument expected")
+            return True
+        else:
```

Comment:
The else statement is unnecessary, you can just return with false at the end of the function.